src/test/java/ApiTestWithJackson.java

```diff
@@ -0,0 +1,66 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.response.Response;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static io.restassured.RestAssured.get;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class ApiTestWithJackson {
+
+    @Test
+    public void getRequestFromGithubTest() throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        String stringURL = "https://api.github.com/users/rusakov66";
+        Response response = get(stringURL);
+        JSONObject jsonResponse = new JSONObject(response.asString());
+        System.out.println(jsonResponse);
```

Comment:
we repeated 100 times about system.out

Reply:
Sorry for SOP's -  there were my debugging points ....